src/sentry_scope.h

```diff
@@ -67,7 +67,8 @@ void sentry__scope_flush_unlock(const sentry_scope_t *scope);
  * attached.
  */
 void sentry__scope_apply_to_event(const sentry_scope_t *scope,
-    sentry_value_t event, sentry_scope_mode_t mode);
+    const sentry_options_t *options, sentry_value_t event,
+    sentry_scope_mode_t mode);
```

Comment:
this formatting offends my rustfmt-trained aesthetics 😄  
but if CI says it's fine I'm not arguing

Reply:
Yes, its apparently the default clang-format style :-( Either way, at least its automatic, and being checked :-D